src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -493,15 +493,15 @@ def martinize(input_pdb, output_path, skipss):
     pdbf_path = os.path.realpath(input_pdb)
     aa_model = p.get_structure("aa_model", pdbf_path)
 
-    # set ALL bfactors to 0
+    # set ALL bfactors to 1
     for model in aa_model:
         for chain in model:
             if chain.id == " ":
                 print("+ ERROR: Empty chain id detected")
                 exit()
```

Comment:
I would rather add a `raise ModuleError` here instead of the `exit()` to terminate the run properly.
requires:
- to import the custom exception at the top of the file:
```python3
from haddock.core.exceptions import ModuleError
```
- to catch this error when calling `martinize()`